### PR TITLE
Allow podman manifest push --purge|-p as alias for --rm

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
podman manifest push -p now alias to --rm for Docker compatibilty
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

File: cmd/podman/utils/alias.go [hunk 1]
```diff
@@ -23,8 +23,6 @@ func AliasFlags(f *pflag.FlagSet, name string) pflag.NormalizedName {
 		name = "ns"
 	case "storage":
 		name = "external"
-	case "purge":
-		name = "rm"
```
Comment (Luap99) [inline cmd/podman/utils/alias.go:27]: Why is this removed? It will effect all commands that use this alias function, i.e. `podman run --purge` would no longer work

Comment (rhatdan) [reply cmd/podman/utils/alias.go:27]: This was added purely for the alias of this command, 
docker run --purge does not exists, so no equivalent to podman run --purge.

If I leave the alias, then the -p shows up in `podman manifest push --help`
I added the alias to specifically fix this problem, so removing it and getting the -p to work as an alias, works better.

Comment (Luap99) [reply cmd/podman/utils/alias.go:27]: Sure I understand why the flag is added. Just saying removing it from the alias will affect other commands and this could break users. I know it is not documented but I see little gain by removing it vs the risk of breaking changes.

Comment (rhatdan) [reply cmd/podman/utils/alias.go:27]: The gain is that cobra is not handling it properly, and is documenting the `-p, rm ...`
Which then requires us to document the -p in the man page, because hack/xref-helpmsgs-manpages complains about it not being documented.
The fact that `podman run --purge` worked, could be considered a bug, I see no reason to make the tool worse just because of the infinitesimally small chance, that someone used an undocumented option. Where said option is not only undocumented for Podman but also for Docker.

Comment (Luap99) [reply cmd/podman/utils/alias.go:27]: mhh, yeah that sounds like a bug in cobra/pflag
If this is required to make it work I am fine with it.

**Comment by Luap99** [comment]
LGTM

**Comment by baude** [comment]
do we not want to update docs?
